Multisig/UI/Safe Management/Create Safe/SafeDeploymentNotificationController.swift

```diff
@@ -57,11 +58,22 @@ class SafeDeploymentNotificationController {
         let known = ["safeCreated", "safeCreationFailed"].contains(type)
         return known
     }
-    
-    static func handleSafeCreatedNotification(userInfo: [AnyHashable : Any]) {
+
+    static func handleSafeCreatedNotification(userInfo: [AnyHashable: Any]) {
         let address: String = userInfo["safe"] as! String
         let chainId: String = userInfo["chainId"] as! String
         let route = NavigationRoute.showAssets(address, chainId: chainId)
         DefaultNavigationRouter.shared.navigate(to: route)
     }
 }
+
+extension String {
+    func ellipsize(maxLength: Int = 14) -> String {
+        if self.count > maxLength {
+            if var shortString = self.substr(0, maxLength) {
```

Comment:
Do you think you could use the method that already exists in the String - self.prefix(maxLength)?